build.sh

```diff
@@ -45,21 +70,32 @@ done
 
 mkdir -p logs
 
+# Check out and patch Mono version
+if [ ! -e ${mono_root} ]; then
+    git clone https://github.com/mono/mono ${mono_root} --branch ${mono_version} --single-branch --progress --depth 1 
+    pushd ${mono_root}
+    if [ ! -z "${mono_commit}" ]; then 
+	git checkout ${mono_commit} 
+    fi
+    git submodule update --init --recursive --recommend-shallow --progress --depth 1 
+    patch -p1 < ${files_root}/patches/mono-unity-Clear-TLS-instead-of-aborting.patch 
+    popd
+fi
+
 export podman_build="$podman build --build-arg img_version=${img_version}"
-export podman_build_mono="$podman_build --build-arg mono_version=${mono_version} --build-arg mono_commit=${mono_commit} -v $(pwd)/files:/root/files"
+export podman_build_mono="$podman_build --build-arg mono_version=${mono_version} -v ${files_root}:/root/files"
 
 $podman build -t godot-fedora:${img_version} -f Dockerfile.base . 2>&1 | tee logs/base.log
 $podman_build -t godot-export:${img_version} -f Dockerfile.export . 2>&1 | tee logs/export.log
 
 $podman_build_mono -t godot-mono:${img_version} -f Dockerfile.mono . 2>&1 | tee logs/mono.log
 $podman_build_mono -t godot-mono-glue:${img_version} -f Dockerfile.mono-glue . 2>&1 | tee logs/mono-glue.log
 $podman_build_mono -t godot-windows:${img_version} -f Dockerfile.windows --ulimit nofile=65536 . 2>&1 | tee logs/windows.log
+
```

Comment:
That's a nitpick but there's no real reason for having an empty line here and not between other builds ;)